articles/active-directory/develop/msal-net-token-cache-serialization.md

```diff
@@ -98,8 +98,8 @@ Here are examples of possible distributed caches:
 services.Configure<MsalDistributedTokenCacheAdapterOptions>(options => 
   {
     // Optional: Disable the L1 cache in apps that don't use session affinity
-    //                 by setting DisableL1Cache to 'false'.
-    options.DisableL1Cache = false;
+    //                 by setting DisableL1Cache to 'true'.
+    options.DisableL1Cache = true;
```

Comment:
```suggestion
    options.DisableL1Cache = false;
```

Reply:
The comment explains when to disable the cache, but the code does not disable it. (disabling it is only for specific scenarios)